test/lib/active_record/connection_adapters/trilogy_adapter_test.rb

```diff
@@ -360,6 +360,64 @@ class ActiveRecord::ConnectionAdapters::TrilogyAdapterTest < TestCase
     end
   end
 
+  class Post < ActiveRecord::Base; end
+
+  test "bulk fixture inserts when multi statement is configured" do
+    ActiveRecord::Base.establish_connection(@configuration.merge(multi_statement: true))
+
+    fixtures = {
+      "posts" => [
+        { "title" => "Foo", "body" => "Something", "kind" => "something else", "created_at" => Time.now.utc, "updated_at" => Time.now.utc },
+      ]
+    }
+
+    assert_nothing_raised do
+      conn = ActiveRecord::Base.connection
```

Comment:
We could probably pull the connection out into a variable at the start of the test (same thing for test below)